### PR TITLE
v8 release

### PR DESCRIPTION
**Pull request by runjin326**
#### What scientific question is your analysis addressing?
Update release note, data description, data format documentations and `download-data.sh` for v8 release.

#### What was your approach?
Include changed files and added description for all of them.

#### What GitHub issue does your pull request address?
https://github.com/PediatricOpenTargets/ticket-tracker/issues/166

#### Which areas should receive a particularly close look?
1) Please see whether all the changes that we made for the release are included in the release note. 
2) Please see whether the data description is detailed enough.
3) Once all the files are uploaded to S3, please also test the download is working properly 

#### Is there anything that you want to discuss further?
No.

### Data Release Checklist
- [x] Is the table in doc/data-file-descriptions.md up to date?
- [x] Is doc/data-format.md up to date?
- [x] Is doc/release-notes.md up to date?
- [x] Is download-data.sh up to date?
- [x] Was download-data.sh tested and did it complete without error?

### CODE REVIEW

File: doc/release-notes.md
Comment (jharenza) [inline doc/release-notes.md]:
```suggestion
      - For `short_histology`,  neuroblastoma samples previously annotated as `NBL` or `Embryonal tumor` are converted to `Neuroblastoma` to be consistent with other samples 
```

File: doc/release-notes.md
Comment (jharenza) [inline doc/release-notes.md]:
```suggestion
      - consensus_wgs_plus_cnvkit_wxs_autosomes.tsv.gz [ticket 159](https://github.com/PediatricOpenTargets/ticket-tracker/issues/159) and [ticket 160](https://github.com/PediatricOpenTargets/ticket-tracker/issues/160) 
      - consensus_wgs_plus_cnvkit_wxs_x_and_y.tsv.gz [ticket 159](https://github.com/PediatricOpenTargets/ticket-tracker/issues/159) and [ticket 160](https://github.com/PediatricOpenTargets/ticket-tracker/issues/160)
```
You can remove these individual files from the md5sum as well, leave on s3, since we have a combined file.

File: doc/release-notes.md
Comment (jharenza) [inline doc/release-notes.md]:
```suggestion
    - Update method to call CNV consensus (WGS) as described in [ticket 134](https://github.com/PediatricOpenTargets/ticket-tracker/issues/134) and [ticket 149](https://github.com/PediatricOpenTargets/ticket-tracker/issues/149). Briefly, CNV called by `MantaSV` were filtered to contain only `filter == "PASS"` before going into the consensus calling workflow. In the subsequent step, instead of only retaining CNV calls that have 50% reciprocal overlap between callers (which was too stringent), the criteria is expanded to include small CNV regions that are 90% covered by a larger CNV. The consensus is the overlapping region. 
```

for the future, please write one sentence per line for easier review :)  

I removed that samples went through two pipelines bc we had that in v7 as well and our filenames/data files description indicate such.

File: doc/release-notes.md
Comment (jharenza) [inline doc/release-notes.md]:
```suggestion
    -As a result of changing consensus calling criteria and adding new TARGET WXS DNA sample results to `cnv-cnvkit.seg.gz` and `cnv-controlfreec.tsv.gz`, the following files were updated:
```

File: doc/release-notes.md
Comment (jharenza) [reply doc/release-notes.md]: For some reason, I cannot change line 8 in release notes:

- deiltaed changes: 
should be - `detailed changes:`

File: doc/release-notes.md
Comment (jharenza) [inline doc/release-notes.md]:
```suggestion
      - Some TARGET WXS samples miss `tumor_ploidy` that are actually available - and those samples now have updated ploidy using the file in the discussion session of [ticket 160](https://github.com/PediatricOpenTargets/ticket-tracker/issues/160)
```

**Review by jharenza — CHANGES_REQUESTED** [review]
few suggestions so far, but all data til now downloaded OK

**Comment by jharenza** [comment]
Your md5sum has a tab instead of two spaces for: 
```
6ccc84316284a15f2cea7fc5cf82e3d5	consensus_wgs_plus_cnvkit_wxs.tsv.gz
```
which prompts there to be an object called `6ccc84316284a15f2cea7fc5cf82e3d5` in the download - can you fix that as well?

**Comment by runjin326** [comment]
> Your md5sum has a tab instead of two spaces for:
> 
> ```
> 6ccc84316284a15f2cea7fc5cf82e3d5	consensus_wgs_plus_cnvkit_wxs.tsv.gz
> ```
> 
> which prompts there to be an object called `6ccc84316284a15f2cea7fc5cf82e3d5` in the download - can you fix that as well?
Yes! Will do that!

File: doc/release-notes.md
Comment (runjin326) [reply doc/release-notes.md]: Fixed!

File: doc/release-notes.md
Comment (runjin326) [reply doc/release-notes.md]: Gotcha! Will do!

File: doc/release-notes.md
Comment (logstar) [inline doc/release-notes.md]:
```suggestion
    - Futher update `ensg-hugo-rmtl-mapping.tsv` [PR 48 D3b codes](https://github.com/d3b-center/D3b-codes/pull/48) to include all gene ENSG ID to symbol mappings in v7 `ensg-hugo-rmtl-mapping.tsv`.
```

Thank you for updating the release notes. Just adding a suggestion to clarify this update.

Comment (runjin326) [reply doc/release-notes.md]: @logstar, thanks so much! I will implement your suggested changes. One quick question - now that ticket 135 is in, do you want to add `all cohort` independent samples in the release as well? If so, which ones?

Comment (logstar) [reply doc/release-notes.md]: Thank you for the question. Yes. It would be great to have `all-cohorts` independent sample lists in the release as well. All corresponding ones for `each-cohort` independent sample lists in slack message <https://opentargetspediatrics.slack.com/archives/C02B5K2B99C/p1629165953031500?thread_ts=1629158109.022400&cid=C02B5K2B99C> should be sufficient. If not, we could add more in v9.

**Comment by runjin326** [comment]
@logstar, @jharenza - I have now updated the release note to contain all the additional changes that we made. Fusion file, independent sample files are also uploaded in S3 with `md5sum.txt` updated as well. Please also check to see the download works.

**Comment by logstar** [comment]
> @logstar, @jharenza - I have now updated the release note to contain all the additional changes that we made. Fusion file, independent sample files are also uploaded in S3 with `md5sum.txt` updated as well. Please also check to see the download works.

Thank you for the updates @runjin326 . The v8 `download-data.sh` works well.

```text
$ head download-data.sh 

set -e
set -o pipefail

# Use the OpenPBTA bucket as the default.
URL=${OPENPBTA_URL:-https://s3.amazonaws.com/kf-openaccess-us-east-1-prd-pbta/open-targets}
RELEASE=${OPENPBTA_RELEASE:-v8}
PREVIOUS=${OPENPBTA_RELEASE:-v7}

# Remove old symlinks in data
$ bash download-data.sh
...
Checking MD5 hashes...
histologies.tsv: OK
cnv-cnvkit.seg.gz: OK
cnv-controlfreec.tsv.gz: OK
consensus_wgs_plus_cnvkit_wxs.tsv.gz: OK
snv-consensus-plus-hotspots.maf.tsv.gz: OK
data-files-description.md: OK
efo-mondo-map.tsv: OK
ensg-hugo-rmtl-mapping.tsv: OK
fusion-arriba.tsv.gz: OK
fusion-putative-oncogenic.tsv: OK
fusion-starfusion.tsv.gz: OK
gene-counts-rsem-expected_count-collapsed.rds: OK
gene-expression-rsem-tpm-collapsed.rds: OK
intersect_cds_lancet_strelka_mutect_WGS.bed: OK
intersect_strelka_mutect_WGS.bed: OK
release-notes.md: OK
sv-manta.tsv.gz: OK
uberon-map-gtex-group.tsv: OK
uberon-map-gtex-subgroup.tsv: OK
WGS.hg38.lancet.300bp_padded.bed: OK
WGS.hg38.lancet.unpadded.bed: OK
WGS.hg38.mutect2.vardict.unpadded.bed: OK
WGS.hg38.strelka2.unpadded.bed: OK
WGS.hg38.vardict.100bp_padded.bed: OK
independent-specimens.rnaseq.primary.eachcohort.tsv: OK
independent-specimens.rnaseq.relapse.eachcohort.tsv: OK
independent-specimens.wgswxspanel.primary.eachcohort.tsv: OK
independent-specimens.wgswxspanel.relapse.eachcohort.tsv: OK
independent-specimens.rnaseq.primary.tsv: OK
independent-specimens.rnaseq.relapse.tsv: OK
independent-specimens.wgswxspanel.primary.tsv: OK
independent-specimens.wgswxspanel.relapse.tsv: OK
```

**Comment by runjin326** [comment]
@jharenza, @logstar, `ensg-hugo-rmtl-mapping.tsv` also uploaded to S3 and `md5sum.txt` updated.

**Review by jharenza — COMMENTED** [review]
@runjin326 - this looks good, but can you update the release notes in the download folder + md5sum? that looks slightly out of date with the release notes in `doc/`. Thanks!

**Comment by runjin326** [comment]
@jharenza, yes I was actually waiting for this PR to be merged but I can do it now!

**Comment by runjin326** [comment]
> @runjin326 - this looks good, but can you update the release notes in the download folder + md5sum? that looks slightly out of date with the release notes in `doc/`. Thanks!

Release note + data file description file update and md5sum matched ;) I checked the download as well and everything looks good!

**Review by jharenza — APPROVED** [review]
>@jharenza, yes I was actually waiting for this PR to be merged but I can do it now!

Typically, we will have all release updates in one PR - once this v8 is merged, we will not want to update any files in that s3 bucket - if they need updates, that would trigger a v9 release. The files look good to me now, so feel free to merge and close the corresponding tickets! Nice work :)

**Pull request merged**
